File: Tournonment/AkhJoonTarafNistS1.java
import java.util.ArrayList;
import java.util.Scanner;

public class Main {

    public static void main(String args[]) {
        Scanner scanner = new Scanner(System.in);

        ArrayList<String> data = new ArrayList<String>() {
            {
                add("shanbe");
                add("1shanbe");
                add("2shanbe");
                add("3shanbe");
                add("4shanbe");
                add("5shanbe");
                add("jome");
            }
        };

        int a = scanner.nextInt();
        for (int i = 0; i < a; i++) {
            data.remove(scanner.next());
        }

        int b = scanner.nextInt();
        for (int i = 0; i < b; i++) {
            data.remove(scanner.next());
        }

        int c = scanner.nextInt();
        for (int i = 0; i < c; i++) {
            data.remove(scanner.next());
        }

        System.out.print(data.size());
    }
}

// <Ctrl+Z>

File: Tournonment/AkhJoonTarafNistS1.py
class Main:
    def main(self):
        data = ["shanbe", "1shanbe", "2shanbe", "3shanbe", "4shanbe", "5shanbe", "jome"]

        a = int(input())
        for _ in range(a):
            data.remove(input())

        b = int(input())
        for _ in range(b):
            data.remove(input())

        c = int(input())
        for _ in range(c):
            data.remove(input())

        print(len(data))

if __name__ == "__main__":
    main = Main()
    main.main()
